modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1328,7 +1106,7 @@ class NDB_Form_User_Accounts extends NDB_Form
         ) {
             // If PHP version < 5.5, use old MD5Unsalt method to check for
             // password change
-            return !User::MD5Unsalt($newPassword, $passwords['Password_hash']);
+            return !\User::MD5Unsalt($newPassword, $passwords['Password_hash']);
```

Comment:
can you make a redmine ticket to delete this? I thought it was all gone.. the password_md5 column doesn't even exist anymore, so I think this line is just a relic of the past.

Reply:
Done RM#13404